apps/wallet-mobile/src/navigation.tsx

```diff
@@ -82,11 +81,11 @@ export const defaultStackNavigationOptions = (theme: Theme): StackNavigationOpti
 
 export const DEPRECATED_defaultStackNavigationOptions: StackNavigationOptions = {
   headerStyle: {
-    backgroundColor: COLORS.BACKGROUND_BLUE,
+    // backgroundColor: COLORS.BACKGROUND_BLUE,
```

Comment:
is it used only in the create/restore funnel? otherwise this change will have side effects in other funnels